Activities/Cryptography/UiPath.Cryptography.Activities/KeyedHashFile.cs

```diff
@@ -33,6 +34,13 @@ public partial class KeyedHashFile : CodeActivity<string>
         [LocalizedDescription(nameof(Resources.Activity_KeyedHashFile_Property_Key_Description))]
         public InArgument<string> Key { get; set; }
 
+
```

Comment:
Make Browsable false to avoid showing it in Studio Desktop

Reply:
I did make Browsable false for the toggle that was added.